grakn-graql/src/main/java/ai/grakn/graql/internal/query/InsertQueryExecutor.java

```diff
@@ -268,8 +247,32 @@ private Concept putConceptByType(Optional<String> name, Optional<String> id, Var
                 Pattern rhs = var.getProperty(RhsProperty.class).get().getRhs();
                 return type.asRuleType().addRule(lhs, rhs);
             });
+        } else if (type.getName().equals(Schema.MetaSchema.CONCEPT.getName())) {
+            throw new IllegalStateException(var + " cannot be an instance of meta-type " + type.getName());
```

Comment:
This is necessary here. I have to switch on all possible type classes to get at each `Type#addInstance` method. The only type that will not have this method is `concept` itself. So, core won't catch that situation.